frappe/patches/v12_0/rename_events_repeat_on.py

```diff
@@ -0,0 +1,27 @@
+import frappe
+from frappe.utils import get_datetime
+
+def execute():
+	weekdays = ["monday", "tuesday", "wednesday", "thursday", "friday", "saturday", "sunday"]
+
```

Comment:
Why not write a single SQL statement to update the values? Doing it in a for loop can take a lot of time.

Reply:
Had to loop through the `Every Week` event so that i could set the proper day to repeat the event